kratos/utilities/math_utils.h

```diff
@@ -845,7 +827,7 @@ class MathUtils
      * @param c The resulting vector
      */
     
-    template< class T1, class T2, class T3 >
+    template< class T1 = array_1d<double, 3>, class T2 = T1, class T3 = T1>
```

Comment:
why?

you could already wrote simply

UnitCrossProduct(a,b,c)

now you can only do if it matches your default

Reply:
just to add that if for any reason the 

       UnitCrossProduct(a,b,c)

does not work, just try

      UnitCrossProduct<>(a,b,c)

Reply:
Done